src/concrete/ml/quantization/quantized_module.py

```diff
@@ -522,14 +522,15 @@ def _fhe_forward(
 
             # If the inference should be executed using simulation
             if simulate:
+                is_crt_encoding = self.fhe_circuit.statistics["packing_key_switch_count"] != 0
 
-                # If the old simulation method should be used
-                if USE_OLD_VL:
+                # Whether the old simulation method should be used based on USE_OLD_VL
```

Comment:
I would rephrase this with : 
```
# If the virtual library method should be used 
# For now, use the virtual library when simulating circuits that uses CRT 
# decomposition because the official simulation is too slow 
# FIXME: https://github.com/zama-ai/concrete-ml-internal/issues/4391
```

Reply:
I don't like this `# If the virtual library method should be used `. Feels like something else is coming to complete the sentence.

Reply:
Anyway back to the original comment with some of your suggestion.